package.json

```diff
@@ -133,6 +133,9 @@
         "ts-node": "^10.5.0",
         "typescript": "^4.5.2"
     },
+    "peerDependencies": {
+        "@swc/helpers": ">0.4.14 <0.5"
```

Comment:
I think we should publish `@swc/helpers@0.5.0`, but this should done by another commit, right?

Reply:
Indeed, I believe this Pull Request does not introduce any breaking changes but rather is a bugfix.

In my opinion, the current helpers are insufficient, and I think it would be worthwhile to introduce breaking changes, including the use of named exports in another Pull Request. I am currently preparing an RFC.